components/src/core/Drawer/DrawerBody.tsx

```diff
@@ -12,64 +14,67 @@ const useStyles = makeStyles({
-                              fontColor: groupProps.fontColor || props.fontColor,
-                              iconColor: groupProps.iconColor || props.iconColor,
-                              onSelect: props.onSelect,
-                              open: props.open,
-                              titleColor: groupProps.titleColor || props.titleColor,
-                          }
-                        : {}
+                if (child.type && child.type.displayName !== 'DrawerNavGroup') return child;
+                const groupProps: DrawerNavGroupProps = child.props;
+
+                // for any DrawerNavGroup, if a prop is not set (undefined), inherit from the DrawerBody
+                // open and onItemSelect is always determined by DrawerBody
+                return (
+                    <DrawerNavGroup
+                        {...groupProps}
+                        key={index.toString()}
+                        activeItem={groupProps.activeItem || bodyProps.activeItem}
+                        activeItemBackgroundColor={
+                            groupProps.activeItemBackgroundColor || bodyProps.activeItemBackgroundColor
+                        }
+                        activeItemFontColor={groupProps.activeItemFontColor || bodyProps.activeItemFontColor}
+                        activeItemIconColor={groupProps.activeItemIconColor || bodyProps.activeItemIconColor}
+                        activeItemBackgroundShape={
+                            groupProps.activeItemBackgroundShape || bodyProps.activeItemBackgroundShape
+                        }
+                        backgroundColor={bodyProps.backgroundColor}
+                        chevron={groupProps.chevron === undefined ? bodyProps.chevron : groupProps.chevron}
+                        collapseIcon={groupProps.collapseIcon || bodyProps.collapseIcon}
+                        divider={groupProps.divider === undefined ? bodyProps.divider : groupProps.divider}
+                        expandIcon={groupProps.expandIcon || bodyProps.expandIcon}
+                        hidePadding={
+                            groupProps.hidePadding === undefined ? bodyProps.hidePadding : groupProps.hidePadding
+                        }
+                        itemFontColor={groupProps.itemFontColor || bodyProps.itemFontColor}
+                        itemIconColor={groupProps.itemIconColor || bodyProps.itemIconColor}
+                        nestedDivider={
+                            groupProps.nestedDivider === undefined ? bodyProps.nestedDivider : groupProps.nestedDivider
+                        }
+                        nestedBackgroundColor={groupProps.nestedBackgroundColor || bodyProps.nestedBackgroundColor}
+                        ripple={groupProps.ripple === undefined ? bodyProps.ripple : groupProps.ripple}
+                        onItemSelect={bodyProps.onItemSelect}
+                        open={bodyProps.open}
```

Comment:
should we rename the `open` prop to `drawerOpen` similar to how we handled the 'item' props? This could be confusing (open seems like it could refer to the open/close state of nested drawer items)